src/dune_rules/assets.ml

```diff
@@ -76,4 +76,4 @@ module V1 : sig
   (** Execute a command and read its output *)
   val run_and_read_lines : string -> string list
 end
-|jbp}
+|jbp}
```

Comment:
Seems like an accidental change.

Reply:
Indeed, I have seen this change on and off while coding. Maybe it's due to the same problem we are debugging here! Anyway, I removed this change from the PR